recipes/rb-open_uri_redirections/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "open_uri_redirections" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: 39e629b006b1d8f90dcc3e76908fc2df36365cb25a8fc525283e57017b2bc04e
+  - url: https://github.com/open-uri-redirections/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 253261aedaa68b6873c0d713385ff9ccf44455cf369199ae81047a13899a65c4
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: true  # [win]
+
+requirements:
+  host:
+    - ruby <2.6
+  run:
+    - ruby <2.6
+
+test:
+  commands:
+    - ruby -r {{ name }} -e 'exit 0'
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: OpenURI patch to allow redirections between HTTP and HTTPS
+  doc_url: https://www.rubydoc.info/gems/open_uri_redirections
+  dev_url: https://github.com/open-uri-redirections/open_uri_redirections
+
+extra:
+  recipe-maintainers:
+    - svengiegerich
```

happy to co-maintain. :)

```suggestion
    - svengiegerich
    - janjagusch
```